CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 * Fix bug in parsing IPv6 addresses. Previously setting a target address or the
   initial upstream address in the config file would result in a malformed value.
   ([PR](https://github.com/hashicorp/boundary/pull/5221)).
+* ui: Fix text color when system mode is dark. ([PR] (https://github.com/hashicorp/boundary-ui/pull/2651)).  
+* ui: Fix memory leak issue when using fetch in oidc authentication. ([PR] (https://github.com/hashicorp/boundary-ui/pull/2640)).
```

Comment:
I think this was only in desktop client but @lisbet-alvarez can confirm.

Reply:
Yup that is correct the oidc memory leak was for desktop client